TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Server.java

```diff
@@ -49,6 +52,9 @@
     private boolean connected;
     private LogStorage logStorage;
     private PublicKey publicKey;
+    private int pid;
+    private final ScheduledExecutorService scheduler;
```

Comment:
(Same here)

Reply:
isn't this already in line 218?
onShutdown